.circleci/config.yml

```diff
@@ -10,28 +10,15 @@ feature_branch: &feature_branch
 version: 2.1
 
 orbs:
-  kubernetes: circleci/kubernetes@0.11.2
-  helm: circleci/helm@1.2.0
-  hmpps: ministryofjustice/hmpps@3.2
+  hmpps: ministryofjustice/hmpps@3.8
+  slack: circleci/slack@4.4
 
-executors:
-  build_executor:
-    docker:
-    - image: mojdigitalstudio/circleci-build-container
```

Comment:
getting rid of this unusual base image that hasn't been updated in 4 years and moving to standard node image